siliconcompiler/core.py

```diff
@@ -4211,6 +4211,18 @@ def _local_process(self, flow, status, steplist, indexlist):
         for (step, index) in flowgraph_nodes:
             # Setting up tool is optional
             self._setup_node(step, index)
+            # Env vars are necessary as test_multiple_tools.py requires it for its version check
+            self._set_env_vars(step, index)
```

Comment:
if this is needed it should go into the `check_tool_version` function, don't forget to remove these variables after this is done

Reply:
Just one test uses this to overwrite the version of a tool.
I think it's probably better to move this into the test instead of `check_tool_version` because the tool version shouldn't depend on the env vars in the normal case.